packages/fluentui/react-builder/src/components/ComponentTree.tsx

```diff
@@ -11,15 +11,41 @@ export type ComponentTreeProps = {
   onCloneComponent?: ({ clientX, clientY }: { clientX: number; clientY: number }) => void;
   onMoveComponent?: ({ clientX, clientY }: { clientX: number; clientY: number }) => void;
   onDeleteComponent?: () => void;
+  onAddComponent?: (uuid: string, where: string) => void;
 };
 
+const isMac = navigator.userAgent.indexOf('Mac OS X') !== -1;
+const behavior = isMac ? treeAsListBehavior : treeBehavior;
+
+const macKeyDown = (e: React.KeyboardEvent<HTMLElement>) => {
+  const keyCode = e.keyCode || e.which;
+  if (e.shiftKey && keyCode === 121) {
+    const activeElement = document.activeElement;
+    if (activeElement) {
+      const event = new MouseEvent('contextmenu', { bubbles: true });
+      activeElement.dispatchEvent(event);
+    }
+  }
+};
+
+const treeKeyDown = isMac ? macKeyDown : undefined;
```

Comment:
If I understand it correctly this undefined will be changed latter, so that it works on other OS as well? Should I add it to the list of task to do on UI builder?

Reply:
no - in other systems this is a default shortcut that is handled by the system. Mac does not have such shortcut unless you start VoiceOver.

Reply:
I see, thanks!